frontend/src/pages/account/index.tsx

```diff
@@ -24,22 +25,22 @@ const Account = () => {
 
   const initialValues = {
     id: null,
-    full_name: '',
+    names: '',
     email: '',
     is_active: false,
     is_superuser: false,
-    affiliation: '',
-    affiliation_type: '',
+    affiliation_organisation: '',
+    affiliation_type: [],
```

Comment:
the CSV you sent me also included "other affiliation"

Reply:
Will look at this later.